components/private/Shapes/tests/unit/ShapesNoopTest.m

```diff
@@ -27,4 +27,23 @@ - (void)testInit {
   XCTAssertNotNil(view);
 }
 
+- (void)testCornerTreatmentEquality {
```

Comment:
Please move this to a new test class. The "noop" test can be removed later since we now actually have tests for shapes.

Reply:
This is the only test file under Shapes directory, the other one I assume you mentioned is for ShapeLibrary/, which is not under Shapes/. I think it makes sense to either rename this class, or create a new test file.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/5954